### PR TITLE
fix(docs): correct the docs for AWS Athena, CLI, DynamoDB and ECR

### PR DESCRIPTION
**Pull request by shrutimantri**

### CODE REVIEW

File: src/main/java/io/kestra/plugin/aws/cli/AwsCLI.java [hunk 1]
```diff
@@ -29,26 +29,20 @@
...
 )
 @Plugin(
         examples = {
                 @Example(
-                        title = "Create a simple S3 bucket",
+                        title = "Create a simple S3 bucket.",
                         code = {
-                                "accessKeyId: \"<access-key>\"",
-                                "secretKeyId: \"<secret-key>\"",
-                                "region: \"eu-central-1\"",
```
Comment (brian-mulier-p) [inline src/main/java/io/kestra/plugin/aws/cli/AwsCLI.java:41]: Please keep that part to show off how credentials should be added

File: src/main/java/io/kestra/plugin/aws/cli/AwsCLI.java [hunk 2]
```diff
@@ -29,26 +29,20 @@
...
                                 "  - aws s3 mb s3://test-bucket"
                         }
                 ),
                 @Example(
-                        title = "List all S3 buckets as the task's output",
+                        title = "List all S3 buckets as the task's output.",
                         code = {
-                                "accessKeyId: \"<access-key>\"",
-                                "secretKeyId: \"<secret-key>\"",
-                                "region: \"eu-central-1\"",
```
Comment (brian-mulier-p) [inline src/main/java/io/kestra/plugin/aws/cli/AwsCLI.java:51]: Please keep that part to show off how credentials should be added

**Review by brian-mulier-p — CHANGES_REQUESTED** [review]

File: src/main/java/io/kestra/plugin/aws/cli/AwsCLI.java (hunk 1 above)
Comment (shrutimantri) [reply src/main/java/io/kestra/plugin/aws/cli/AwsCLI.java:41]: This is not present in most other AWS services. 
I am planning to add an "Authentication" section on this page: https://kestra.io/plugins/plugin-aws similar to the Authentication section present for GCP here: https://kestra.io/plugins/plugin-gcp

File: src/main/java/io/kestra/plugin/aws/cli/AwsCLI.java (hunk 2 above)
Comment (shrutimantri) [reply src/main/java/io/kestra/plugin/aws/cli/AwsCLI.java:51]: This is not present in most other AWS services.
I am planning to add an "Authentication" section on this page: https://kestra.io/plugins/plugin-aws similar to the Authentication section present for GCP here: https://kestra.io/plugins/plugin-gcp

**Comment by shrutimantri** [comment]
As discussed, I have added accessKeyId, secretKeyId and region in all the examples.

**Review by brian-mulier-p — APPROVED** [review]
LGTM

**Pull request merged**